features/utils/jenkins.py

```diff
@@ -0,0 +1,44 @@
+import requests
+import subprocess
+
+
+def setCredential(url, username, password,  credential_id, key, value, credfile="userpass_cred.sh"):
+    downloadjenkinsJar(url)
+    jenkins_cli_command = "java -jar jenkins-cli.jar -auth %s:%s -s  http://%s" % (
+                          username, password, url)
+    subprocess.check_call(
+        [
+            "bash",
+            "utils/"+credfile,
+            "%s" % (jenkins_cli_command),
+            credential_id,
+            key,
+            value
+            ])
+
+
+def startJob(url, username, password, jobUrl):
```

Comment:
`requests.post(url)`, not `curl`, please

Reply:
Done